### DISCUSSION TITLE
Customized answers for local vault based on online LLM

### DISCUSSION THREAD

**Discussion by wwjCMP** [logancyang/obsidian-copilot#370] [Ideas]
Usually, online LLM can provide an acceptable answer to a problem, but the problem is that they are not customized enough. Specifically, it is not closely related to one's own valut. Here is a workflow I envisioned

![image](https://github.com/logancyang/obsidian-copilot/assets/32979859/15e45d3e-c676-446a-bf5e-f6ff51666597)

**Comment by logancyang** [comment]
I'm not following. Chat in Discord?

**Comment by wwjCMP** [comment]
> I'm not following. Chat in Discord?

OK
